tsdb/raw.go

```diff
@@ -264,6 +265,19 @@ func (e *RawExecutor) execute(out chan *models.Row) {
 			e.limitTagSet(chunkedOutput.key())
 			continue
 		}
+		// Check to see if our client disconnected, or it has been to long since
+		// we were asked for data...
+		select {
+		case <-closing:
+			out <- &models.Row{Err: fmt.Errorf("execute was closed by caller")}
+			break
+		case <-timeout:
+			// This should never happen, so if it does, it is a problem
+			out <- &models.Row{Err: fmt.Errorf("execute was closed by caller")}
```

Comment:
@benbjohnson they don't get logged, as there is no logger in play.  I could simply log to stderr though.  Thoughts?